### PR TITLE
Igor/exercise states

### PR DESCRIPTION
**Pull request by igormas**

### CODE REVIEW

File: src/components/infraWebAppZnkExercise/directives/znkExerciseHeader/exerciseHeader.template.html [hunk 1]
```diff
@@ -3,17 +3,8 @@
...
-            <div class="math-no-calc"></div>
-            <svg-icon name="math-icon" class="icon-wrapper"></svg-icon>
-
-            <md-tooltip md-visible="vm.showToolTip" ng-if="vm.showToolTip" md-direction="bottom" class="no-calc-tooltip md-whiteframe-3dp">
-                <div class="arrow-up"></div>
-                <div translate=".NO_CALC_TOOLTIP" class="top-text"></div>
-                <div translate=".GOT_IT" class="md-button primary got-it-btn" ng-click="vm.showToolTip = false"></div>
-            </md-tooltip>
+        <div class="side-text">
+            {{vm.sideText | cutString: 40}}
```
Comment (assafshp) [inline src/components/infraWebAppZnkExercise/directives/znkExerciseHeader/exerciseHeader.template.html]: shouldn't we leave:
<div class="side-text" translate="{{vm.sideText | cutString: 40}}" translate-values="{subjectName: vm.subjectName, exerciseNum: vm.exerciseNum}"></div>
this is generic

File: src/components/znkExerciseStatesUtility/controllers/infraWebAppExerciseState.controller.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                this.isComplete = isExerciseComplete;
+                // tutorial intro
+                // if (exerciseData.exerciseTypeId === ExerciseTypeEnum.TUTORIAL.enum) {
+                //     if (angular.isArray(exerciseData.exercise.content)) {
+                //         angular.forEach(exerciseData.exercise.content, function (content) {
+                //             content.title = content.title.replace(/font\-family: \'Lato Regular\';/g, 'font-family: Lato;font-weight: 400;');
+                //             content.body = content.body.replace(/font\-family: \'Lato Regular\';/g, 'font-family: Lato;font-weight: 400;');
+                //         });
+                //     }
+                //
```
Comment (assafshp) [inline src/components/znkExerciseStatesUtility/controllers/infraWebAppExerciseState.controller.js]: ? needed?

File: src/components/infraWebAppZnkExercise/directives/znkExerciseHeader/exerciseHeader.template.html (hunk 1 above)
Comment (igormas) [inline src/components/infraWebAppZnkExercise/directives/znkExerciseHeader/exerciseHeader.template.html]: Its no calc icon which related only to sat

File: src/components/znkExerciseStatesUtility/controllers/infraWebAppExerciseState.controller.js (hunk 2 above)
Comment (igormas) [inline src/components/znkExerciseStatesUtility/controllers/infraWebAppExerciseState.controller.js]: Yep

**Pull request merged**